scipy/special/_basic.py

```diff
@@ -3156,3 +3157,46 @@ def zeta(x, q=None, out=None):
         return _ufuncs._riemann_zeta(x, out)
     else:
         return _ufuncs._zeta(x, q, out)
+
+      
+def softplus(x, **kwargs):
+    r"""
+    Compute the softplus function element-wise.
+
+    The softplus function is defined as: ``softplus(x) = log(1 + exp(x))``.
+    It is a smooth approximation of the rectifier function (ReLU).
+
+    Parameters
+    ----------
+    x : array_like
+        Input array-like object.
+
+    **kwargs :
+        out: Specify the output array.
```

Comment:
`axis` and `axes` aren't keywords, and this won't render properly. I suggest copying exactly how `np.logaddexp` documents this, so it looks and renders like a ufunc.